### PR TITLE
AN-5214 fix. Profile pictures are uploaded using only v3 

### PR DESCRIPTION
**Pull request by makingthematrix**
v2 caused error responses: Bad request(400).

### CODE REVIEW

**Comment by deanrobertcook** [comment]
if it's going to go into the release, please make a hotfix and rebase to master :)

File: zmessaging/src/main/scala/com/waz/sync/SyncServiceHandle.scala [hunk 1]
```diff
@@ -173,7 +173,7 @@ class AccountSyncHandler(zms: Signal[ZMessaging], otrClients: OtrClientsSyncHand
     case SyncUser(u)                           => zms.usersSync.syncUsers(u.toSeq: _*)
     case SyncSearchQuery(query)                => zms.usersearchSync.syncSearchQuery(query)
     case SyncRichMedia(messageId)              => zms.richmediaSync.syncRichMedia(messageId)
-    case DeletePushToken(token)                 => zms.gcmSync.deleteGcmToken(token)
+    case DeletePushToken(token)                => zms.gcmSync.deleteGcmToken(token)
```
Comment (deanrobertcook) [inline zmessaging/src/main/scala/com/waz/sync/SyncServiceHandle.scala]: unnecessary change

File: zmessaging/src/main/scala/com/waz/model/UserInfo.scala [hunk 2]
```diff
@@ -170,7 +169,6 @@ object UserInfo {
       info.name.foreach(o.put("name", _))
       info.accentId.foreach(o.put("accent_id", _))
       info.picture.foreach(ps => o.put("assets", encodeAsset(ps)))
-      info.picture.foreach(ps => o.put("picture", encodePicture(ps)))
```
Comment (deanrobertcook) [inline zmessaging/src/main/scala/com/waz/model/UserInfo.scala]: You can also delete the `encodePicture` method I guess

File: zmessaging/src/main/scala/com/waz/sync/SyncServiceHandle.scala (hunk 1 above)
Comment (makingthematrix) [inline zmessaging/src/main/scala/com/waz/sync/SyncServiceHandle.scala]: reverted

File: zmessaging/src/main/scala/com/waz/model/UserInfo.scala (hunk 2 above)
Comment (makingthematrix) [inline zmessaging/src/main/scala/com/waz/model/UserInfo.scala]: OK, done.
